app/controllers/catalog_controller.rb

```diff
@@ -57,8 +57,8 @@ class CatalogController < ApplicationController
     config.view.gallery.partials = [:index_header, :index]
     config.view.masonry.partials = [:index]
     config.view.slideshow.partials = [:index]
-    config.view.maps.type = 'placename_coord'
-    config.view.maps.placename_coord_field = 'placename_coords_ssim'
+    config.view.maps.type = 'bbox'
```

Comment:
I couldn't find any indication we've ever indexed anything into `placename_coords_ssim`, so using the srpt field which we ostensibly index into seemed like an improvement.